code/__HELPERS/unique_ids.dm

```diff
@@ -26,7 +26,7 @@ var/global/next_unique_datum_id = 1
 		tag = tag_backup
 	return unique_datum_id
 
-/client/proc/UID()
+/client/UID()
```

Comment:
don't override this completely; just get rid of it, otherwise, there's going to be problems.